demo/src/app/app.config.ts

```diff
@@ -139,4 +139,8 @@ export class AppConfig extends AbstractAppConfig {
   public getWorkAllocationApiUrl(): string {
     return this.config.work_allocation_api_url;
   }
+
+  public getUserInfoApiUrl(): string {
```

Comment:
Really not understanding why this is an addition... it's already on `master`, isn't it?